### PR TITLE
View all trucks

### PR DESCRIPTION
**Pull request by moorea**
The work this PR includes is self-explanatory in the title. 

Still not sure about the logo for the button or the placement. Let me know what you think. I don't think it's great, but I don't have a better idea at the moment.

### CODE REVIEW

File: TruckMuncher/AllTrucksCollectionViewController.swift [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    let trucksManager = TrucksManager()
+    var menuManager = MenuManager()
+    
+    init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: NSBundle?, allTrucks: [RTruck]) {
+        self.allTrucks = allTrucks
+        super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
+    }
+
+    required init(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
```
Comment (jault3) [inline TruckMuncher/AllTrucksCollectionViewController.swift]: Just call the super.init(coder: aDecoder) method here

File: TruckMuncher/Constants.swift [hunk 2]
```diff
@@ -32,5 +32,7 @@ let PROPERTIES_FILE = "Properties"
 let BASE_URL = "https://api.truckmuncher.com:8443"
 #elseif DEBUG
 let PROPERTIES_FILE = "Properties-dev"
-let BASE_URL = "http://truckmuncher:8443"
+let BASE_URL = "https://api.truckmuncher.com:8443"
```
Comment (jault3) [inline TruckMuncher/Constants.swift]: Change this back

File: TruckMuncher/TruckDetailView.swift [hunk 3]
```diff
@@ -97,7 +117,7 @@ class TruckDetailView: UIView, UITableViewDataSource, UITableViewDelegate {
     
     // MARK - UITableViewDelegate and UITableViewDataSource Methods
     func numberOfSectionsInTableView(tableView: UITableView) -> Int {
-        return Int(menu!.categories.count)
+        return menu != nil ? Int(menu!.categories.count) : 0
```
Comment (jault3) [inline TruckMuncher/TruckDetailView.swift]: This is what I choose to swiftify for this PR :smile: `return menu?.categories.count ?? 0`

**Comment by jault3** [comment]
Very minor issues, overall looks great. Just fix those couple items noted above and :shipit:

File: TruckMuncher/AllTrucksCollectionViewController.swift (hunk 1 above)
Comment (moorea) [inline TruckMuncher/AllTrucksCollectionViewController.swift]: :skull: errors are better

**Pull request merged**